planning/behavior_path_planner/src/marker_util/debug_utilities.cpp

```diff
@@ -361,4 +361,49 @@ MarkerArray createObjectsMarkerArray(
 
   return msg;
 }
+
+MarkerArray createDrivableLanesMarkerArray(
+  const std::vector<DrivableLanes> & drivable_lanes, std::string && ns)
+{
+  MarkerArray msg;
+
+  int32_t i{0};
+
+  const auto get_lanelet_marker =
+    [&ns, &i](const auto & lanelet, const auto r, const auto g, const auto b) {
+      auto marker = createDefaultMarker(
+        "map", rclcpp::Clock{RCL_ROS_TIME}.now(), ns, bitShift(lanelet.id()) + i++,
+        Marker::LINE_STRIP, createMarkerScale(0.1, 0.0, 0.0), createMarkerColor(r, g, b, 0.999));
+
+      if (!lanelet.polygon3d().empty()) {
+        marker.points.reserve(lanelet.polygon3d().size() + 1);
+      }
+
+      for (const auto & p : lanelet.polygon3d()) {
+        marker.points.push_back(createPoint(p.x(), p.y(), p.z()));
+      }
+
+      if (!marker.points.empty()) {
+        marker.points.push_back(marker.points.front());
+      }
```

Comment:
```suggestion
      if (lanelet.polygon3d().empty()) {
        return marker;
      }

      marker.points.reserve(lanelet.polygon3d().size() + 1);

      for (const auto & p : lanelet.polygon3d()) {
        marker.points.push_back(createPoint(p.x(), p.y(), p.z()));
      }

      marker.points.push_back(marker.points.front());
```

I think return `marker` early if `polygon3d()` is `empty` reduces `if` statement.
What are your thoughts on this?

Reply:
I totaly agree with you. I'll fix it.